backend/go.mod

```diff
@@ -27,7 +27,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
-require github.com/coreos/go-oidc/v3 v3.11.0
+require (
```

Comment:
What happened here with these extra lines?

Reply:
It got automatically added when I did `go mod tidy`